### PR TITLE
docs: Convert READMEs to Markdown

### LINKED ISSUE #226: Docs: convert all rst readmes to markdown
Add readmes should just use markdown for consistency and developer experience.

### PR DESCRIPTION
**Pull request by anticorrelator**
resolves #226

### CODE REVIEW

File: python/instrumentation/openinference-instrumentation-openai/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# OpenInference OpenAI Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-openai.svg)](https://pypi.org/project/openinference-instrumentation-openai/)
+
+## Installation
+
+    pip install openinference-instrumentation-openai
```
Comment (axiomofjoy) [inline python/instrumentation/openinference-instrumentation-openai/README.md:7]: Needs code block.

File: python/instrumentation/openinference-instrumentation-llama-index/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# OpenInference LlamaIndex Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-llama-index.svg)](https://pypi.org/project/openinference-instrumentation-llama-index/)
+
+## Installation
+
+    pip install openinference-instrumentation-llama-index
```
Comment (axiomofjoy) [inline python/instrumentation/openinference-instrumentation-llama-index/README.md:7]: Needs code block

File: python/instrumentation/openinference-instrumentation-langchain/README.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# OpenInference LangChain Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-langchain.svg)](https://pypi.org/project/openinference-instrumentation-langchain/)
+
+## Installation
+
+    pip install openinference-instrumentation-langchain
```
Comment (axiomofjoy) [inline python/instrumentation/openinference-instrumentation-langchain/README.md:7]: Needs code block

File: python/instrumentation/openinference-instrumentation-dspy/README.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# OpenInference DSPy Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-dspy.svg)](https://pypi.org/project/openinference-instrumentation-dspy/)
+
+## Installation
+
+    pip install openinference-instrumentation-dspy
```
Comment (axiomofjoy) [inline python/instrumentation/openinference-instrumentation-dspy/README.md:7]: Needs code block

File: python/instrumentation/openinference-instrumentation-bedrock/README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# OpenInference AWS Bedrock Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-bedrock.svg)](https://pypi.org/project/openinference-instrumentation-bedrock/)
+
+## Installation
+
+    pip install openinference-instrumentation-bedrock
```
Comment (axiomofjoy) [inline python/instrumentation/openinference-instrumentation-bedrock/README.md:7]: Needs code block

**Review by axiomofjoy — APPROVED** [review]

File: python/instrumentation/openinference-instrumentation-bedrock/README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# OpenInference AWS Bedrock Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-bedrock.svg)](https://pypi.org/project/openinference-instrumentation-bedrock/)
+
+## Installation
+
+    `pip install openinference-instrumentation-bedrock`
```
Comment (mikeldking) [inline python/instrumentation/openinference-instrumentation-bedrock/README.md:7]:
```suggestion
    ```shell
    pip install openinference-instrumentation-bedrock
    ```
```

File: python/instrumentation/openinference-instrumentation-langchain/README.md [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# OpenInference LangChain Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-langchain.svg)](https://pypi.org/project/openinference-instrumentation-langchain/)
+
+## Installation
+
+    `pip install openinference-instrumentation-langchain`
```
Comment (mikeldking) [inline python/instrumentation/openinference-instrumentation-langchain/README.md:7]:
```suggestion
    ```shell
    pip install openinference-instrumentation-langchain
    ```
```

File: python/instrumentation/openinference-instrumentation-openai/README.md [hunk 8]
```diff
@@ -0,0 +1,7 @@
+# OpenInference OpenAI Instrumentation
+
+[![pypi](https://badge.fury.io/py/openinference-instrumentation-openai.svg)](https://pypi.org/project/openinference-instrumentation-openai/)
+
+## Installation
+
+    `pip install openinference-instrumentation-openai`
```
Comment (mikeldking) [inline python/instrumentation/openinference-instrumentation-openai/README.md:7]:
```suggestion
    ```shell
    pip install openinference-instrumentation-openai
    ```
```

**Review by mikeldking — APPROVED** [review]

**Pull request merged**